ARAnalytics.m

```diff
@@ -420,7 +433,7 @@ + (void)event:(NSString *)event {
 }
 
 + (void)event:(NSString *)event withProperties:(NSDictionary *)properties {
```

Comment:
I didn't see, currently a proper way to implement this in our code base, I can maybe use swizzling, so that every time:withProperties is called, it will add the localized hour
@zats what do you think?